modules/appnexusBidAdapter.js

```diff
@@ -1120,9 +1124,13 @@ function buildNativeRequest(params) {
  * @param {string} elementId element id
  */
 function hidedfpContainer(elementId) {
-  var el = document.getElementById(elementId).querySelectorAll("div[id^='google_ads']");
-  if (el[0]) {
-    el[0].style.setProperty('display', 'none');
+  try {
+    var el = document.getElementById(elementId).querySelectorAll("div[id^='google_ads']");
```

Comment:
Since you are making changes here anyway, can this be updated to use a const instead of a var?